selenium_webdriver/src/test/java/tests/sauceDemoTests/SauceDemoLoginButtonErrorMessageTest.java

```diff
@@ -0,0 +1,24 @@
+package tests.sauceDemoTests;
+
+import model.User;
+import org.testng.annotations.Test;
+import pageobject.sauceDemoPages.SauceDemoLoginPage;
+import service.UserCreator;
+import tests.BaseTest;
+
+import static org.testng.Assert.assertEquals;
+
+public class SauceDemoLoginButtonErrorMessageTest extends BaseTest {
+
+    @Test
+    public void getLoginButtonErrorMessage() {
+        User testUser = UserCreator.withCredentialsFromProperty();
```

Comment:
it's unclear from Test that we deal with Locked User

Reply:
Fixed